readme.md

```diff
@@ -20,13 +20,16 @@ console.log(await publicIp.v4());
 
 console.log(await publicIp.v6());
 //=> 'fe80::200:f8ff:fe21:67cf'
+
+console.log(await publicIp())
+//=> will return Ipv6 if you have one, otherwise will return Ipv4
```

Comment:
This should contain an example output. This explaination should be in a comment above the line.

Reply:
fixed at [ae1b80](https://github.com/sindresorhus/public-ip/pull/59/commits/ae1b80dc08bdd1accc5935d5641f08ff6bbaed8a).

However, I'm not sure if it is clear enough what the function does. Anyways the explanation is in the API section, a few lines below

Reply:
You'll still need to add a comment above the call.